dumpling/install.sh

```diff
@@ -15,3 +15,11 @@ chmod a+x bin/minio
 
 wget https://dl.minio.io/client/mc/release/linux-amd64/mc -O bin/mc
 chmod a+x bin/mc
+
+go get github.com/ma6174/snappy
```

Comment:
```suggestion
```

only need go install?

Reply:
error encountered in CI integration test

https://github.com/pingcap/tidb/actions/runs/3444134903/jobs/5746413416#step:6:7138

i think maybe should back `go get github.com/ma6174/snappy`?

Reply:
error encountered in CI integration test

https://github.com/pingcap/tidb/actions/runs/3444134903/jobs/5746413416#step:6:7138

i think maybe should add back `go get github.com/ma6174/snappy`?